lightblue-latest/docker-compose.yml

```diff
@@ -13,3 +11,10 @@ mongodb:
   ports:
    - "28017:28017"
    - "27017:27017"
+applications:
+  image: lightblue/applications
+  command: /opt/jbossas7/bin/standalone.sh -b 0.0.0.0
```

Comment:
Your documentation says this will be available to localhost but you're binding to 0.0.0.0.  Is this intentional?  If it should only be localhost then I expect bind to 127.0.0.1 in all cases jboss is started.